src/pages/copy-of-index.jsx

```diff
@@ -0,0 +1,33 @@
+import Head from "next/head";
+import React from "react";
+import Link from "next/link";
+import Button from "../atoms/button";
+import useGet from "../ions/hooks/fetch/get";
+import Layout from "../organisms/layout";
+
+const Page = () => {
+	const { data, loading, error } = useGet("/api/hello");
+
+	return (
+		<Layout>
+			<Head>
+				<title key="title">My Project</title>
+				<meta key="description" name="description" content="This is my project" />
+			</Head>
+			<h1>Home</h1>
+			{loading && <div>Loading...</div>}
+			{error && <div>{error.message}</div>}
+			{data && (
+				<pre>
+					<code>{JSON.stringify(data, null, 4)}</code>
+				</pre>
+			)}
+			<Button>Click me</Button>
+			<Link href="/foobar">
+				<a>Foobar</a>
+			</Link>
+		</Layout>
+	);
+};
+
+export default Page;
```

Comment:
I assume this file is only a backup!? If so, you can delete it.

Reply:
Yes, it is my Spickzettel! :)